.swiftlint.yml

```diff
@@ -120,6 +120,7 @@ excluded: # paths to ignore during linting. Takes precedence over `included`.
   - ThirdParty
   - UITests/
   - XCUITests/
+  - l10n-screenshots-dd/
```

Comment:
@electricRGB I'm adding this folder path here as there are swiftlint rules that fail in files that we can't change